software/otterwiki.yml

```
@@ -0,0 +1,12 @@
+name: "An Otter Wiki"
```

Matches the name of the file and the GitHub repo.
```suggestion
name: "Otter Wiki"
```